llm_memory/components/embedding_provider.py

```
@@ -927,19 +928,33 @@ async def _get_embeddings_with_batch(
                     self._validate_embedding_count(result, texts)
                     return result
                 else:
-                    # 分批并发处理
-                    tasks = []
-                    for i in range(0, len(texts), batch_size):
-                        batch = texts[i : i + batch_size]
-                        task = provider.get_embeddings(batch)
-                        tasks.append(task)
+                    # 分批并发处理，但限制同时请求的批次数量，避免触发 429。
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)